internal/api/attestationconfigapi/client.go

```diff
@@ -36,16 +37,17 @@ func NewClient(ctx context.Context, cfg staticupload.Config, cosignPwd, privateK
 	}
 
 	repo := &Client{
-		s3Client:      s3Client,
-		s3ClientClose: clientClose,
-		signer:        sigstore.NewSigner(cosignPwd, privateKey),
-		bucketID:      cfg.Bucket,
+		s3Client:        s3Client,
+		s3ClientClose:   clientClose,
+		signer:          sigstore.NewSigner(cosignPwd, privateKey),
+		bucketID:        cfg.Bucket,
+		cacheWindowSize: versionWindowSize,
```

Comment:
Make this an argument to the constructor.

Reply:
where do you want to store the constant then? attestationconfigapi.DefaultVersionWindowSize and then rely that the user finds this sensible default? That doesn't seem so user friendly to me